src/main/java/nextstep/fp/Lambda.java

```diff
@@ -26,28 +27,30 @@ public void run() {
         }).start();
     }
 
-    public static int sumAll(List<Integer> numbers) {
+    public static int sumAll(List<Integer> numbers, Conditional conditional) {
         int total = 0;
         for (int number : numbers) {
-            total += number;
+            if(conditional.test(number)){
+                total += number;
+            }
         }
         return total;
     }
 
-    public static int sumAllEven(List<Integer> numbers) {
+    public static int sumAllEven(List<Integer> numbers, Conditional conditional) {
         int total = 0;
         for (int number : numbers) {
-            if (number % 2 == 0) {
+            if(conditional.test(number)){
                 total += number;
             }
         }
         return total;
     }
 
-    public static int sumAllOverThree(List<Integer> numbers) {
+    public static int sumAllOverThree(List<Integer> numbers, Conditional conditional) {
```

Comment:
```suggestion
    public static int sum(List<Integer> numbers, Conditional conditional) { }
```

위와 같이 sum과 같은 메서드를 추가하고 sumAll, sumAllEven, sumAllOverThree 3개 메서드의 중복을 제거해 보는 경험이 이 미션의 목표이다.